### DISCUSSION TITLE
2k video streaming

### DISCUSSION THREAD

**Discussion by b-love-dev** [ChristopherKlay/StadiaEnhanced#73] [General]
Hi, I'm unable to get cyberpunk to stream at 2k on my 2k monitor. I get vp9 at 1080p when I enable 2k streaming. My internet connection is 30mbit down. Could that be the issue? Thanks in advance for the help.

**Comment by ChristopherKlay** [comment]
1. Did you go into Stadias settings to select "Up to 4K" once?
2. 30mbit can definitely result in hitting limits with 2K where the game will have issues and/or revert down to 1080p, yes.

**Comment by b-love-dev** [comment]
I recently tried setting stadia enhanced to settings 1440p (2k), but after applying the settings and then reloading the game with the 2k settings, stadia enhanced monitor reports the game is running at 1080p and encoding with vp9, so I hit shift + tap and click connection. Before I click on the 4k setting it says to enable 4k you need to reopen the game, so I click on the 4k setting, then right click on chrome and left click on reload to reopen the game. The game reloads and I press pressing shift tab again and click connection. The 4k setting still says I need to reload the game to apply the 4k setting, so I left click to apply the 4k settings and then right click on chrome to reload cyberpunk 2077.  Stadia enhanced monitor appears and reports the game is running at 1080p with vp9 encoding. So I'm thinking the problem is with my bandwidth. I did select a speed test afterwards through stadia enhanced and it reported 29.753 mbit down.

**Comment by ChristopherKlay** [comment]
Yea, even 1080p can take up most of that in a busy (i.e. high contrast) moment.

As a sidenote however, when Stadia says to "reload" the game, it doesn't mean refreshing the page - which doesn't actually reload/restart the game. You sometimes have to hold the "esc" key and close the game and then start it again; only doing so actually restarts the game.
